tests/models/abiotic_simple/test_microclimate.py

```diff
@@ -5,12 +5,10 @@
 from xarray import DataArray
 
 
-def test_log_interpolation(
-    dummy_climate_data, fixture_core_components, fixture_empty_array
-):
+def test_log_interpolation(dummy_climate_data, fixture_core_components):
     """Test interpolation for temperature and humidity non-negative."""
 
-    from virtual_ecosystem.models.abiotic_simple.microclimate import log_interpolation
+    from virtual_ecosystem.models.abiotic_simple.microclimate_2 import log_interpolation
```

Comment:
Why microclimate_2?

Reply:
Bugger - was testing the new implementation in parallel because I kept screwing it up, and forgot to revert that switch once I'd aligned them!